test/unit/lookups/olleh_test.rb

```diff
@@ -238,7 +250,7 @@ def test_olleh_addr_step_search
   end
 
   def test_olleh_result_wgs_coordinates
-    VCR.use_cassette('address_step_search_plus_coordinates') do
+    VCR.use_cassette('address/address_step_search_plus_coordinates') do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.